.github/workflows/actions/build-core/action.yml

```diff
@@ -1,12 +1,12 @@
-name: 'Build Ionic Core'
-description: 'Build Ionic Core'
+name: "Build Ionic Core"
+description: "Build Ionic Core"
 inputs:
   ionicons-version:
-    description: 'The NPM tag of ionicons to install.'
+    description: "The NPM tag of ionicons to install."
     type: string
     required: false
 runs:
-  using: 'composite'
+  using: "composite"
```

Comment:
Can we revert the quotation marks change?

Reply:
Done!